Makefile

```diff
@@ -43,7 +44,7 @@ ci-install:
 	sudo dpkg -i /tmp/pachctl.deb
 	pip install tox tox-travis
 
-ci-setup:
+ci-setup: 
```

Comment:
```suggestion
ci-setup:
```

Reply:
Removing the extra `rm -rf` above meant GitHub couldn't apply this :-(. Added it in a followup commit